regulations/static/regulations/js/source/views/comment/comment-index-view.js

```diff
@@ -36,6 +36,13 @@ module.exports = Backbone.CommentIndexView = Backbone.View.extend({
   render: function() {
     var html = this.template({comments: this.parseComments()});
     this.$index.html(html);
+
+    if (this.parseComments().length) {
```

Comment:
* We probably don't need to call `parseComments` twice in this method--we can write something like `var comments = this.parseComments()`, and then use `comments` on line 37 and in this conditional.
* We can cache the `$('.comment-index-review')` selection in `initialize` so that we don't have to look it up again on each render. Also, it's probably safer to write `this.$el.find('.comment-index-review')`, just in case there's another element with that class outside this view (shouldn't happen, but you never know).